omero/sysadmins/unix/install-web/web-deployment.txt

```diff
@@ -1,5 +1,5 @@
-TRIAL: OMERO.web WSGI deployment
-================================
+OMERO.web WSGI deployment
```

Comment:
I'm still concerned that the distinction between this page and the deployment with Nginx and Gunicorn page isn't clear, can we add something about how this is for deploying **separately** to OMERO.server to the title? "Deploying OMERO.web separately to OMERO.server using WSGI"? or something similar?